sympy/concrete/summations.py

```diff
@@ -269,9 +269,54 @@ def _eval_difference_delta(self, n, step):
 
         return Sum(f, (k, upper + 1, new_upper)).doit()
 
-    def _eval_simplify(self, ratio, measure):
-        from sympy.simplify.simplify import sum_simplify
-        return sum_simplify(self)
+    def _eval_expand_mul(self, **hints):
```

Comment:
I'm skeptical that this is the right function name: `_eval_expand_mul` should be flattening products of sums into sums of products, not factoring. I would recommend either changing the name to a different private function.